gulp/smart-agents.js

```diff
@@ -0,0 +1,36 @@
+const gulp = require('gulp')
+const path = require('path')
+const fs = require('fs')
+
+gulp.task('link-agents', () => {
```

Comment:
did you tested this? Looks like gulp 3 syntax...

Reply:
new Syntax looks like `gulp.task('link-agents', gulp.series(() => { //... `

Reply:
> new Syntax looks like `gulp.task('link-agents', gulp.series(() => { //... `

Bug in generator, ticket has been created...